src/transport/SessionManager.cpp

```diff
@@ -497,9 +497,10 @@ Optional<SessionHandle> SessionManager::AllocateSession(SecureSession::Type secu
     return mSecureSessions.CreateNewSecureSession(secureSessionType, sessionEvictionHint);
 }
 
-CHIP_ERROR SessionManager::InjectPaseSessionWithTestKey(SessionHolder & sessionHolder, uint16_t localSessionId, NodeId peerNodeId,
-                                                        uint16_t peerSessionId, FabricIndex fabric,
-                                                        const Transport::PeerAddress & peerAddress, CryptoContext::SessionRole role)
+CHIP_ERROR SessionManager::InjectPaseSessionWithTestKey(Crypto::SessionKeystore & keystore, SessionHolder & sessionHolder,
```

Comment:
Suggest using the fact that SessionManager has a SessionKeystore to avoid needing the extra arg